web/modules/custom/dbcdk_community/src/Plugin/Block/ProfilesBlock.php

```diff
@@ -253,6 +280,13 @@ protected function parseProfile(Profile $profile, array $columns) {
           ]);
           break;
 
+        // Generate community profile link based on a pattern defined on the
+        // settings page.
+        case 'community_link':
+          $community_site_url = Url::fromUri($this->urlGenerator->generate($profile));
+          $row[] = Link::fromTextAndUrl($this->t('Biblo.dk'), $community_site_url);
```

Comment:
We should agree on a standard. You decide which you prefer.

About url/link generation is calling these static methods the way to go? [It seems like the way to go compared to the complexity of using an injected LinkGenerator](https://github.com/DBCDK/biblo-admin/blob/develop/web/modules/custom/dbcdk_community/src/Plugin/Block/FlaggedContentList.php#L172-L175). These classes do not seem to hook into other parts of the Drupal architecture anyway.